README.md

```diff
@@ -121,6 +122,21 @@ _note: if .dropOffDate is not set, it will be calculated with .pickUpDate + .dur
 |         | .dateFrom    | '20092017' 
 |         | .dateTo      | '20092017' 
 
+
+| type      | fields       | example
+| :---      | :---         | :---
+|'roundTrip'| .bookingId   | 'NEZE2784NQXTHEN' 
+|           | .destination | 'YYZ' 
+|           | .no          | '1' 
```

Comment:
we should give that property a more readable name - what does this property mean - number of what?

Reply:
As you suggested, I have written an email to afu@nezasa.com to find out, what this mystery things is.

Reply:
:+1: